content/user/ui/report/Vulnerability-trends.md

```diff
@@ -0,0 +1,71 @@
+
+###Filter vulnerabilities
+
+Each report defaults to all applications, servers and rules, but you can filter vulnerabilities by clicking in the fields above the graph. The following table outlines the categories that you can use to create a custom report. 
+
+
+| Field        | Default      | Filter Options                                                                                              |
+|--------------|--------------|-------------------------------------------------------------------------------------------------------------|
+| Date         | Last 7 Days  | Last 30 days <br> Last 12 weeks <br> Last 12 months                                                                   |
+| Applications | All          | Importance (Critical, High, Medium, Low, Unimportant) <br> Application Tags <br> Licensed (List of all applications) |
+| Servers      | All          | Environment (Development, QA, Production) <br> Server Tags <br> Servers (List of all servers)                         |
+| Rules        | All          | Severity (Critical, High, Medium, Low, Note) <br> Vulnerability Tags <br> Vulnerability Rules (List of all rules)                         |
+
+
+<!-- Add info re: build number option - App > Vuln trend -->
+
+## Save Reports
+
+You can save filter criteria to recall any customized report at a later time. Saved reports are at the User level, so each of you have your own defined list of saved vulnerability trend reports. You can edit or delete these reports at any time. 
+
+<a href="assets/images/vulnerability-trend-report-name.png" rel="lightbox" title="Save a report"><img class="thumbnail" src="assets/images/vulnerability-trend-report-name.png"/></a> 
+ 
+To save a report view, click the star icon at the top right of the graphs. This generates a popup with a field to name the report. Once saved, the named report appears next to the **Vulnerability Trend** heading with a dropdown menu. Each time you come to the Vulnerability Trend page, the menu shows all of your saved reports as well as an option to **Start a new report**. 
+
+<a href="assets/images/Vulnerability-trend-save-report.png" rel="lightbox" title="Menu of saved reports"><img class="thumbnail" src="assets/images/Vulnerability-trend-save-report.png"/></a> 
+
+### Rename reports
+
+When viewing a saved report, hover over the star icon to generate a **Manage Report** tooltip. Click the icon to produce a popup with a field to rename the report and buttons to **Cancel**, **Remove** or **Save**. 
+
+### Edit and remove reports
+
+If you change filter options while viewing a saved report, the star icon appears and the report name is updated as **Edited**. Click the icon to generate a popup menu to **Save Existing** or **Save As New**. Choose **Save Existing** to update the saved report name with the current filters and remove the **Edited** status. Choose **Save As New** to save the report view with the current filters as a new report under a different name.
```

Comment:
"...the star icon will change to an unsaved state and an "Edited" label will appear next to the report name. "